ros_buildfarm/templates/snippet/builder_system-groovy_check-for-failing-jobs.xml.em

```diff
@@ -0,0 +1,82 @@
+@(SNIPPET(
+    'builder_system-groovy',
+    command=
+"""// CHECK FOR FAILING JOBS
+import hudson.model.Job
+
+// Control the number of failures in a row to check
+min_number_of_failures = 5
+// Control the minimun amount of time failing
+min_days_failing = 7
```

Comment:
@mikaelarguedas @tfoote @wjwwood  Since you are in charge of at least one ROS distro which thresholds (in line 8 and 10) make sense from your point of view?

@mikaelarguedas Can you please look at the output of the current job (http://build.ros.org/job/lunar_failing-jobs/1/console) and check if the result makes sense.